Makefile

```
@@ -46,7 +46,11 @@ vet:
 
 # Run go test against code
 test:
-	go test ./...
+	go test ./apis/...
+
+# Run conformance tests against controller implementation
+conformance:
+	go test ./conformance/...
```

```suggestion
	go test ./conformance/... -v
```

Otherwise it just hangs with no info about what is happening

---

Updated, thanks!